Postman Test Suite/Ed-Fi ODS-API Profile Test Suite.postman_collection.json

```diff
@@ -10057,17 +10120,24 @@
 									"listen": "test",
 									"script": {
 										"exec": [
-											"\r",
-											"pm.environment.set(\"known:school:PrimaryKeys\", getPrimaryKeys(\"edFi_school\"));\r",
-											"pm.environment.set(\"known:studentSpecialEducationProgramAssociations:PrimaryKeys\", getPrimaryKeys(\"edFi_studentSpecialEducationProgramAssociation\")\r",
-											"    .concat(\"educationOrganizationReference\",\"programReference\",\"studentReference\"));\r",
+											"if (pm.response.code == 404) {\r",
```

Comment:
Rather than rely on processing Swagger metadata, these fixtures will now fall back to using hard-coded, well-known and stable values.